src/main/resources/templates/directory.html

```diff
@@ -15,7 +15,26 @@
     </nav>
 </header>
 <main th:fragment="bodycontent">
-    <!-- Restlichen Inhalt hier einfügen! -->
+    <table>
+        <tr>
+            <th>Name</th>
+            <th>Datum</th>
+            <th>Typ</th>
+            <th>Größe</th>
+        </tr>
+        <tr th:each="dir : ${dirs}"} th:href="@{/${dir.id}}">
+            <td th:text="${dir.directoryName}">%Name%</td>
+            <td></td>
+            <td>Dateiordner</td>
+            <td></td>
+        </tr>
```

Comment:
folders should be above files.

Reply:
They are - There is one table which shows folders and files in that order